moto/ssm/responses.py

```diff
@@ -253,6 +253,10 @@ def put_parameter(self):
             name, description, value, type_, allowed_pattern, keyid, overwrite, tags
         )
 
+        if tags:
```

Comment:
This logic should be part of the `put_parameter`-method in models.py, for a few reasons:

1) The model should take care of the formatting/logic of extracting tags
2) Storing every property of a parameter should be in a single place/method
2) The `put_parameter` call might fail. Having the logic here could result in tags being added regardless, and still receiving the ParameterAlreadyExists-error

Reply:
Done